Develop the UI of the calculator

sambeck87 opened the pull request:
- Create a directory called components.
- Inside components create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design
- Develop just the UI of the calculator.
- Use class-based components.

sambeck87 committed:
Develop the UI of the calculator

tresorsawasawa commented on src/logo.svg:
```diff
@@ -1 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3"><g fill="#61DAFB"><path d="M666.3 296.5c0-32.5-40.7-63.3-103.1-82.4 14.4-63.6 8-114.2-20.2-130.4-6.5-3.8-14.1-5.6-22.4-5.6v22.3c4.6 0 8.3.9 11.4 2.6 13.6 7.8 19.5 37.5 14.9 75.7-1.1 9.4-2.9 19.3-5.1 29.4-19.6-4.8-41-8.5-63.5-10.9-13.5-18.5-27.5-35.3-41.6-50 32.6-30.3 63.2-46.9 84-46.9V78c-27.5 0-63.5 19.6-99.9 53.6-36.4-33.8-72.4-5 ...
```
- [ ] As required for the project, kindly make sure you delete all the boilerplate from CRA (text, images, styles,...). You should also delete the `reportWebVitals.js`, `setupTests.js`,... Make sure you delete all the non related files of your project.

tresorsawasawa commented on src/App.js:
```diff
@@ -1,11 +1,9 @@
 function App() {
   return (
     <div className="App">
       <header className="App-header">
```
- [ ] Kindly make sure the the `App.js` main component file is a `class-based` and not a `function-based` as required for the project.

tresorsawasawa commented on src/components/Calculator.js:
```diff
@@ -0,0 +1,33 @@
+import '../App.css';
+import React from 'react';
+// eslint-disable-next-line
```
- [ ] Kindly make sure you fix your `eslint` errors instead of disabling them for best practices, clean code and for the project requirements

sambeck87 replied:
@tresorsawasawa Hi.
I have a problem with that linter, that linter asks me for a function instead of a class that's why I  disabled that linter (well I understood that)
![Captura desde 2023-01-09 20-10-32](https://user-images.githubusercontent.com/113067895/211446339-2ca5d7ef-008e-4163-8249-e8f013507aaa.png)


May you help me with a suggestion to fix this problem, please?

tresorsawasawa replied:
Hi @sambeck87

Sorry to answer later, I was off. And right now I am not working, I am so far away from working place. But once I get home I can see what to do

sambeck87 replied:
Thanks, I have solved the problem.

tresorsawasawa commented on src/App.js:
```diff
@@ -1,11 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
 
 function App() {
   return (
```
- [ ] Kindly make sure the `App.js`(main component) has the imported `Calculator.js` component for the project requirements i.e. you should first import the `Calculator.js` component in the `App.js` file then after import the `App.js` (main component) in the `index.js` file(not directly import) `Calculator.js` in the `index.js` file
e.g.:
````JavaScript
//  In the App.js file:
import Calc from './components/Calculator';
...

//  In the index.js file:
import App from './App.js';


const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
);
```

tresorsawasawa reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @sambeck87 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sambeck87 committed:
Delete all the boilerplate from CRA
Base all in class-based
Enable all linters
Import calculator component to App component and App component to index,js file

youmari reviewed (approved):
# Project Approved :trophy: :tada: 🟢 

Hi @sambeck87 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
